remoteappmanager/application.py

```diff
@@ -181,11 +191,43 @@ def _reverse_proxy_init(self):
             self._config.proxy_api_url
         ))
 
+    def _db_init(self):
+        """Initializes the database connection."""
+        self.db = orm.Database(self.config.db_url)
+
+    def _user_init(self):
+        """Initializes the user at the database level."""
+        session = self.db.create_session()
+
+        with transaction(session):
+            try:
+                user = session.query(orm.User).filter_by(
+                    name=self.config.user).one_or_none()
+            except MultipleResultsFound:
+                self.log.error("Multiple results found when "
+                               "querying for username {}. This is supposedly "
+                               "impossible because the username should be a "
+                               "unique key by design.".format(
+                                   self.config.user))
+                raise
+
+            if user is None:
+                user = orm.User(name=self.config.user)
+                session.add(user)
+
+            # make sure that the user always has at least one team: his own.
+            if len(user.teams) == 0:
+                team = orm.Team(name=self.config.user)
+                user.teams.append(team)
+                session.add(team)
```

Comment:
Similar comment to the one above, if we are sure of adding unfound user to the database, `session.commit` after `session.add`